nexus/db-queries/src/db/datastore/switch_port.rs

```diff
@@ -451,19 +451,18 @@ impl DataStore {
                     .load_async::<SwitchPortLinkConfig>(&conn)
                     .await?;
 
-                let lldp_svc_ids: Vec<Uuid> = result
+                let lldp_link_ids: Vec<Uuid> = result
                     .links
                     .iter()
-                    .map(|link| link.lldp_service_config_id)
+                    .map(|link| link.lldp_link_config_id)
                     .collect();
 
-                use db::schema::lldp_service_config as lldp_config;
-                use db::schema::lldp_service_config::dsl as lldp_dsl;
-                result.link_lldp = lldp_dsl::lldp_service_config
-                    .filter(lldp_config::id.eq_any(lldp_svc_ids))
-                    .select(LldpServiceConfig::as_select())
+                use db::schema::lldp_link_config;
+                result.link_lldp = lldp_link_config::dsl::lldp_link_config
+                    .filter(lldp_link_config::id.eq_any(lldp_link_ids))
```

Comment:
If we want to use soft-deletion, this should also probably:

`.filter(lldp_link_config::dsl::time_deleted.is_null())`

If we want to use hard-deletion, this isn't necessary, but then we should drop the `time_deleted` column